app/services/shelf_list.rb

```diff
@@ -12,20 +12,20 @@ def self.call(args)
     new(args).build
   end
 
-  attr_reader :query, :forward_limit, :reverse_limit, :forward_shelfkey, :reverse_shelfkey
+  attr_reader :query, :forward_limit, :reverse_limit, :classification
 
   def initialize(
     query:,
     forward_limit:,
     reverse_limit:,
-    forward_shelfkey: 'forward_lc_shelfkey',
-    reverse_shelfkey: 'reverse_lc_shelfkey'
+    classification:
   )
     @query = query
     @reverse_limit = (reverse_limit || 10).to_i
     @forward_limit = (forward_limit || 10).to_i
     @forward_shelfkey = forward_shelfkey
     @reverse_shelfkey = reverse_shelfkey
```

Comment:
you don't need the forward_shelfkey and reverse_shelfkey attr_readers or these lines above if you have the methods below.

Reply:
removed